src/style.css

```
@@ -0,0 +1,148 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+}
+
+h2,
+h3,
+p {
+  font-size: 0;
+  margin-block-start: 0;
+  margin-block-end: 0;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card {
+  width: 198px;
```

Add main padding to card from mock-up
```suggestion
  width: 198px;
  pading: 32px 16px 16px
```